### DISCUSSION TITLE
project

### DISCUSSION THREAD

**Discussion by AKSHITHA-CHILUKA** [recodehive/Opensource-practice#44] [Q&A] [answered]
what kind of topics should I choose for my project

**Comment by tusharkhakare** [comment, marked as answer]
Pick something you’re stoked about,
maybe AI ethics, renewable energy, or game dev. 
Solve a real problem, keep it doable, and make it stand out.

**Comment by Ali-Hassan2** [comment]
@AKSHITHA-CHILUKA  You should pick topics that align with your own interests and skills it’s easier to stay motivated when you work on something you actually enjoy. Also, look for projects that solve a real problem, even a small one, so your contribution feels meaningful. For example, in AI you could look into projects like:

AI-powered document verification systems (OCR + NLP)

Chatbots and personal AI assistants (using LLMs like GPT, LLaMA, etc.)

Fake news or spam detection using NLP

AI-based recommendation systems (movies, jobs, learning resources)

Healthcare AI like symptom checkers or medical image analysis (open datasets available)

**Comment by Pritamx4** [comment]
choose a fully ai powered health care dashboard for the gym persons
